homepage/homepage/content/docs/upgrade/0-20-0.mdx

```diff
@@ -1,71 +1,97 @@
 import { Alert, CodeGroup } from '@/components/forMdx'
 
-# Jazz 0.20.0 - Crypto provider cleanup (RNCrypto default)
+# Jazz 0.20.0 - Full native crypto
 
-This release removes the PureJSCrypto and QuickCrypto implementations and makes **`RNCrypto` (Rust, via `cojson-core-rn`) the default crypto provider on React Native**
-and **`WasmCrypto` is the default crypto provider for Web/SSR/Workers (wasm only)**.
+With this release, we complete our migration to a pure Rust toolchain.
+We have removed the JavaScript compatibility layer, meaning our native Rust core now runs everywhere: React Native, Edge runtimes, all server-side environments, and the web.
 
-So you can use only:
-- WASM for the web and edge runtimes
-- RNCrypto for React Native
-- Node-API or WASM for Node.js
+## Motivation
 
-## What changed
+Before 0.20.0, Jazz relied on a JavaScript crypto implementation to work around compatibility issues in some runtimes—most notably React Native.
+As a result, our native Rust core wasn’t running in React Native and, on some Edge runtimes, 
+only worked when WASM was explicitly initialized (see: https://jazz.tools/docs/react/server-side/setup#wasm-on-edge-runtimes).
 
-### React Native / Expo
+This JavaScript path was also slower than native Rust crypto. Even RNQuickCrypto (a React Native provider) couldn’t match native performance because it only wrapped some native crypto libraries,
+ rather than running Jazz’s full Rust crypto implementation.
```

Comment:
```suggestion
The JavaScript crypto implementation is much slower than native Rust crypto. Although workarounds like RNQuickCrypto for React Native improved performance, they still only wrapped certain native libraries, rather than running Jazz's full Rust crypto.
```

Wording nit